src/atoms/Layout/index.js

```diff
@@ -170,6 +172,10 @@ Layout.propTypes = {
    */
   fullwidth: PropTypes.bool,
   /**
+   * removes left and right padding from Layout.
```

Comment:
IMO this should be clearer -- "prevents `Layout` from passing left and right padding to all of its children (`<Cols />`). Will not remove padding that is explicitly added to a `<Col />`. << This should also be true if it's not.